lib/http/HttpClient_Apple.mm

```diff
@@ -50,10 +57,7 @@ void SendAsync(IHttpResponseCallback* callback)
         {
             m_callback = callback;
             NSString* url = [[NSString alloc] initWithUTF8String:m_url.c_str()];
-            NSURLSessionConfiguration* sessionConfig = [NSURLSessionConfiguration defaultSessionConfiguration];
             m_urlRequest = [[NSMutableURLRequest alloc] initWithURL:[NSURL URLWithString:url]];
-            m_session = [NSURLSession sessionWithConfiguration:sessionConfig];
-            m_session.sessionDescription = url;
```

Comment:
We don't need to set `sessionDescription` anymore?

Reply:
The session Description is just a string used for debugging purposes. Previously since we had created new URL session each time we made a network call, we set the description to the URL. But now since it is a static URL Session having a session description set to one URL won't be of much use. Hence removed it.

Removing it will not cause any issues to the workflow. Apple documentation for the same: https://developer.apple.com/documentation/foundation/nsurlsession/1408277-sessiondescription